digits/model/images/job.py

```diff
@@ -2,6 +2,10 @@
 from __future__ import absolute_import
 
 from ..job import ModelJob
+from digits.utils import subclass, override
+
+import json
+import os
```

Comment:
I have been contaminated by @lukeyeager and will reproduce his comment that PEP8 recommends putting standard package imports first ([see](https://www.python.org/dev/peps/pep-0008/#imports)) :-)